components/Document/DocumentOptions.tsx

```diff
@@ -20,13 +25,14 @@ import {
   faEllipsis,
   faArrowDownToBracket,
 } from "@fortawesome/pro-regular-svg-icons";
-import { StyleSheet, css } from "aphrodite";
+import { StyleSheet } from "aphrodite";
 import colors from "~/config/themes/colors";
 import { useContext } from "react";
 import { DocumentContext } from "./lib/DocumentContext";
 import useCacheControl from "~/config/hooks/useCacheControl";
-import removeDocFromFeatured from "../Admin/api/removeDocFromFeaturedAPI";
 import excludeFromFeed from "../Admin/api/excludeDocFromFeedAPI";
+import censorDocument from "./api/censorDocAPI";
+import { useAlert } from "react-alert";
```

Comment:
Also — don't need to change this now — but I find this alert design kinda ugly and unaligned with our site design.

Reply:
I don't love it either. We should look into globally updating the style of it. I'll add to our backlog